source/admin_manual/lua.rst

```diff
@@ -26,9 +26,11 @@ reusing scripts by keeping track of which scripts are loaded. Each script has
 it's own memory pool, which is guaranteed to be released when script is
 unloaded.
 
-When script is loaded, :func:`script_load` function is called if found. This can
+When script is loaded, :func:`script_init` function is called if found. This can
 return non-zero to indicate that the script has a problem.
```

Comment:
Is this still true? I somehow remember differently.

Reply:
Yeah, it's not true anymore. Should replace this saying that errors should be raised with error()

Reply:
this is going to 2.3 though